CCDB/src/CCDBDownloader.cxx

```diff
@@ -395,11 +424,14 @@ int CCDBDownloader::startTimeout(CURLM* multi, long timeout_ms, void* userp)
 void CCDBDownloader::setHandleOptions(CURL* handle, PerformData* data)
 {
   curl_easy_setopt(handle, CURLOPT_PRIVATE, data);
-
   curl_easy_setopt(handle, CURLOPT_CLOSESOCKETFUNCTION, closesocketCallback);
   curl_easy_setopt(handle, CURLOPT_CLOSESOCKETDATA, this);
   curl_easy_setopt(handle, CURLOPT_OPENSOCKETFUNCTION, opensocketCallback);
   curl_easy_setopt(handle, CURLOPT_OPENSOCKETDATA, this);
+
+  curl_easy_setopt(handle, CURLOPT_TIMEOUT_MS, mRequestTimeoutMS);
+  curl_easy_setopt(handle, CURLOPT_CONNECTTIMEOUT_MS, mConnectionTimeoutMS);
```

Comment:
so with this, curl with automatically time out when no answer received within a certain time?